docs/user/packaging.rst

```diff
@@ -0,0 +1,93 @@
+Packaging
+=========
+
+You can package a whole trained Fonduer pipeline model (parsing, extraction, featurization, and classification) and deploy it to a remote place to serve.
+To this end, we use `MLflow Model`_ as a storage format.
+A packaged Fonduer pipeline model (or simply referred to as a Fonduer model) looks like this:
+
+    .. code-block::
+        :caption: Directory written by fonduer.packaging.save_model
+
+        fonduer_model/
+        ├── MLmodel
+        ├── code
+        │   ├── my_subclasses.py
+        │   └── my_fonduer_model.py
+        ├── conda.yaml
+        ├── candidate_classes.pkl
+        ├── mention_classes.pkl
+        └── model.pkl  # the pickled Fonduer pipeline model.
+
+Currently, two types of classifiers are supported: :class:`EmmentalModel` (aka discriminative model) and :class:`LabelModel` (aka generative model).
+The following example shows how to package a Fonduer pipeline model that uses :class:`EmmentalModel` as a classifier.
+
+Example
+-------
+
+First, create a class that inherits :class:`FonduerModel` and implements :func:`_classify`.
+Then, put this class in a Python module like `my_fonduer_model.py` instead of in a Jupyter notebook or a long Python script as this module will be packaged.
```

Comment:
remove `long`

Reply:
I'll remove it.

Reply:
Resolved at 1fd13d11c61ac6b27a49bc72a8bcad9808725449.